package/MDAnalysis/analysis/align.py

```diff
@@ -506,9 +506,9 @@ def alignto(mobile, reference, select="all", weights=None,
         try:
             # treat subselection as AtomGroup
             mobile_atoms = subselection.atoms
-        except AttributeError:
+        except AttributeError as e:
```

Comment:
We prefer variable names with multiple letters such as `err` as it is a lot easier to search for.